src/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -35,6 +35,14 @@ public T CreateFake<T>(Action<IFakeOptions<T>> optionsBuilder)
             return (T)this.fakeAndDummyManager.CreateFake(typeof(T), options => optionsBuilder((IFakeOptions<T>)options));
         }
 
+        public object CreateFake(Type typeOfFake, Action<IFakeOptions> optionsBuilder)

```

Comment:
I realized I didn't copy the documentation from the interface. It's an internal class, so I'm not sure it really matters, but if you feel it's important, I'll add it.